project/pom.xml

```diff
@@ -119,6 +119,24 @@
             <artifactId>gson</artifactId>
             <version>2.8.6</version>
         </dependency>
+
+        <dependency>
```

Comment:
duplicate

Reply:
> Very pretty code

Thank you :)

> duplicate dependency

Thanks, nice catch, I'll merge and then fix this dependency in the next PR